scss/_buttons.scss

```diff
@@ -60,6 +59,11 @@
     }
   }
 
+  .btn-check:focus:not(:focus-visible) + &,
+  &:focus:not(:focus-visible) {
+    outline: 0;
```

Comment:
Shouldn't we go with `outline-color: transparent` instead, to improve high contrasts mode support ?

Reply:
oh, do we do this elsewhere already? otherwise, i'd say let's do it consistently, but in a separate PR

Reply:
answering my own question: no, seems we're using `outline: 0` everywhere at the moment. so yes, I'd say let's park that thought, and then do it consistently throughout in a new PR specifically for WHCM improvements